source/extensions/transport_sockets/tls/cert_validator/spiffe_validator.h

```diff
@@ -0,0 +1,77 @@
+#pragma once
```

Comment:
Structurally this validator should be in a separate directory, and build with `envoy_cc_extension`.